dirs/dirs.go

```diff
@@ -140,6 +141,13 @@ var (
 	FeaturesDir string
 )
 
+// User defined home directory variables
+// Not exported, use SnapHomeDirs() and SetSnapHomeDirs() instead
+var (
+	snapHomeDirsMu sync.Mutex
+	snapHomeDirs   []string
```

Comment:
this should start has []string{"/home"} or SnapHomeDirs should return that if this is empty?